platform/tests/metrics/amplifier-prometheus_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run --rm --network ampnet appcelerator/amptools curl -sf http://amplifier:5100/metrics
```

Comment:
amptools is quite big, unless it's already present on the VM, it would be faster to use appcelerator/alpine:3.6.0, it also has curl.